metals/src/main/scala/scala/meta/internal/tvp/ClasspathTreeView.scala

```diff
@@ -33,7 +34,8 @@ class ClasspathTreeView[Value, Key](
       viewId,
       rootUri,
       title + s" (${toplevels().size})",
-      collapseState = MetalsTreeItemCollapseState.collapsed
+      collapseState = MetalsTreeItemCollapseState.collapsed,
+      contextValue = "root"
```

Comment:
Why do we need the new `contextValue` ? This should be documented in tree view protocol https://scalameta.org/metals/docs/integrations/tree-view-protocol if we add it here.

Reply:
It's so VSCode can show a context menu only on projects, not libraries, in the treeview.  If you're ok with that then I'll add some docs

Reply:
Would we be able to do it in VS Code only? I would rather avoid changing it if it's not needed for any other extensions. @ckipp01 what do you think?

Or maybe, we could make it general and add `command` filed instead something like:
```
"comand" :  { "id" : "show-target", args: [] }
```
what do you think?

Reply:
I think that is a cool option to have it under right click, just wondering if we can make the API change less VS Code reliant. I am also worried that with this change we will need to update all the clients.

Reply:
Actually, let me take a look if we can do it without the added parameter

Reply:
If at all possible I think we should avoid baking in VS Code specific stuff to the actual protocol itself. The more we do that the harder it becomes to work around for all other clients. I don't really understand the full context behind this to be honest. I don't know what a context menu is.

Reply:
ContextValue is gone.  The context menu allows you to right-click on a project in the treeview and get a menu up to allow you to call Display the Build Target info directly on the project/target, instead of running the Display Build Target command and getting a list of project/targets up.  Just a different way of getting to view project info